pipeline/lib/current_bench_json.ml

```diff
@@ -69,18 +74,19 @@ module V2 = struct
     | Floats of float list
     | Assoc of (string * float) list
 
+  type line_range = int list option
```

Comment:
Couple of questions because I expected `type line_range = int * int` or `{ start : int ; stop : int }`:
- Why is there an `option`?
- Can the list contains more or less than two elements?

Reply:
The `option` me using a `list` instead of a `int * int` was an artefact from the older code and me not thinking clearly.  I've now removed the option and `type line_range = int * int` as expected.